docs/develop/go/versioning.mdx

```diff
@@ -212,80 +204,24 @@ w := worker.New(c, "your_task_queue_name", workerOptions)
-   Operation: &client.BuildIDOpAddNewIDInNewDefaultSet{
-      BuildID: "deadbeef",
-   },
-})
-```
-
-This code adds the `deadbeef` Build ID to the Task Queue as the sole version in a new version set, which becomes the default for the queue.
-New Workflows execute on Workers with this Build ID, and existing ones will continue to process by appropriately compatible Workers.
-
-If, instead, you want to add the Build ID to an existing compatible set, you can do this:
-
-```go
-// ...
-err := client.UpdateWorkerBuildIdCompatibility(ctx, &client.UpdateWorkerBuildIdCompatibilityOptions{
-   TaskQueue: "your_task_queue_name",
-   Operation: &client.BuildIDOpAddNewCompatibleVersion{
-      BuildID:                   "deadbeef",
-      ExistingCompatibleBuildId: "some-existing-build-id",
-   },
-})
-```
-
-This code adds `deadbeef` to the existing compatible set containing `some-existing-build-id` and marks it as the new default Build ID for that set.
-
-You can also promote an existing Build ID in a set to be the default for that set:
-
-```go
-// ...
-err := client.UpdateWorkerBuildIdCompatibility(ctx, &client.UpdateWorkerBuildIdCompatibilityOptions{
-   TaskQueue: "your_task_queue_name",
-   Operation: &client.BuildIDPromoteIDWithinSet{
-      BuildID: "some-existing-build-id",
-   },
-})
-```
+Importantly, if you start this Worker, it won't receive any tasks until you set up assignment rules.
+See the [Pre-release README.md](https://github.com/temporalio/temporal/blob/main/docs/worker-versioning.md) for more information.
 
-You can also promote an entire set to become the default set for the queue. New Workflows will start using that set's default.
+### Specify versions for Activities, Child Workflows, and Continue-as-New Workflows
 
-```go
-// ...
-err := client.UpdateWorkerBuildIdCompatibility(ctx, &client.UpdateWorkerBuildIdCompatibilityOptions{
-   TaskQueue: "your_task_queue_name",
-   Operation: &client.BuildIDPromoteSet{
-      BuildID: "some-existing-build-id",
-   },
-})
-```
-
-### Specify versions for Commands
-
-By default, Activities, Child Workflows, and Continue-as-New use the same compatible version set as the Workflow that invoked them if they're also using the same Task Queue.
+By default, Activities, Child Workflows, and Continue-as-New Workflows stay on the build of the workflow that spawned them if they are also being run on the same task queue.
+If they are being run on a separate task queue, they default to using the current assignment rules.
 
 If you want to override this behavior, you can specify your intent via the `VersioningIntent` field on the appropriate options struct.
 
 {/* For more information refer to the [conceptual documentation](/workers#worker-versioning). */}
 
-For example, if you want to use the latest default version for an Activity, do the following inside your Workflow code:
+For example, if you want an activity to use the latest assignment rules rather than inheriting from its parent, do:
```

Comment:
s/activity/Activity/

I think you can skip ", do" and just colon after parent.